roles/ostree_repositories/tasks/main.yml

```diff
@@ -5,3 +5,4 @@
     description: Centos Atomic Build Repository
     baseurl: http://buildlogs.centos.org/centos/7/atomic/x86_64/Packages/
     gpgcheck: no
+  when: no_ostree_repo is not defined
```

Comment:
It'd be better to solve this at a higher level and not include the role when not it's not needed

Reply:
I get that, but how :)
in katello/meta/main.yml
```
dependencies:
...
  - role: ostree_repositories
```
is there some conditional dependencies in ansible?

Reply:
An alternative would be to set the repo to enabled/disabled. That means you still include the role, but also disable any potentially active repo. It'd probably be named something like `ostree_repositories_enabled` and default to `yes`, allowing you to override it with `no` to disable.

Reply:
Out of curiosity, where are you getting ostree from if not this repository? That may help us craft the solution.

Reply:
well, I might have miss something, but i disable it and i manage to install a working katello server. I deploy using kickstart, and i do not see where the centos atomic repo could add value to my setup.
Regards
Mike

Reply:
since you initially brought that up, we've merged support for a meta role that sets up the repositories: `foreman_server_repositories` and you can turn off OSTree by passing `foreman_server_repositories_ostree: false` in your playbook now. Does that fix it for you?